### PR TITLE
Enable syncing of contacts & style pairing section

### LINKED ISSUE #6638: [Pairing] restore sync of contact list across devices
### Description

*Type*: Feature

*Summary*: after merge https://github.com/status-im/status-react/pull/6362 feature of syncing contacts between pairing devices was removed (https://github.com/status-im/status-react/pull/6242) and should be restored.

#### Expected behavior
contact list is synced across devices

#### Actual behavior
contact list is not synced

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

### Step to tests

1) You need to have enabled development mode
2) Go to settings -> Devices
3) On android/ios click on `+`, desktop click on `pair` to advertise your device
4) Any other device should see the new device with a 3-words random name + device-type under `devices`
5) If you enable the device, it will send 1-to-1 & group messages to that device, make sure that devices are enabled both ways

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: nightly 07/11/2018
* Operating System: Android, iOS, Mac OSx

### PR DESCRIPTION
**Pull request by cammellos**
fixes #6638

Summary:
Restore syncing of contacts for paired devices.

This will only sync contacts on a one-off basis, it will not keep them in sync.

Partially implements the designs, to enabled/disable one of the devices, click on the row.

Icons on desktop are not looking great, will try to fix, but if I don't make it, will go in a different PR.

status: ready

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

**Review by janherich — APPROVED** [review]

File: src/status_im/ui/screens/pairing/styles.cljs [hunk 1]
```diff
@@ -1,27 +1,78 @@
 (ns status-im.ui.screens.pairing.styles
```
Comment (flexsurfer) [inline src/status_im/ui/screens/pairing/styles.cljs:1]: could you please fix styles for this file, map values alignments

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/pairing/styles.cljs (hunk 1 above)
Comment (cammellos) [reply src/status_im/ui/screens/pairing/styles.cljs:1]: done!

**Comment by yenda** [comment]
`whisper-id` was wiped from the codebase, please use `public-key` instead

**Comment by churik** [comment]
@cammellos 
testing is still WIP, but so far:
**1. On desktop cannot see device type**
Steps:
1) I installed build on IOS, desktop, and Android
2) Tapped on "Pair this device" on Android
3) Checked Devices on IOS, desktop

On IOS I can see 3-random name + Icon
On desktop  3-random name + Unknown label.
Would be nice at least to have platform name instead of Unknown

**2. Last synced contact is appearing without icon (desktop)**

Steps:
1) installed build on IOS, desktop, and Android
2) Enabled sync on three devices
3) Add new contact on Android, Tap on Sync all devices
4) Check contact list on desktop

Expected result: all contacts are synced with default pictures
Actual result: contact is shown without icon
desk_3
Log: [Status.log](https://github.com/status-im/status-react/files/2572304/Status.log)

**3. 1-1 chat is not synced**

1) installed build on IOS, desktop, and Android
2) Enabled sync on three devices
3) Send several messages to 1-1 chat on IOS

Expected result: all messages sent from IOS are visible on Android and Mac OS
Actual result: outgoing messages are not synces

**Comment by cammellos** [comment]
@churik I wasn't able to get a working build, jenkins seems unhappy, the best I can get is this https://ci.status.im/blue/organizations/jenkins/status-react%2Fpull%20requests/detail/PR-6658/26/pipeline/43 ( no Linux), I also had a look at the tests, it seems it's stuck on creating account, I have tested the builds and both creating a new account and recovery seem to be working ok, so maybe are false negatives?

<blockquote><div><strong><a href="https://ci.status.im/blue/organizations/jenkins/status-react%2Fpull%20requests/detail/PR-6658/26/pipeline/43">Jenkins</a></strong></div></blockquote>

**Comment by cammellos** [comment]
closing as included in #6697

**Pull request closed without merging**